BACnet.csproj

```diff
@@ -71,7 +75,17 @@
     <Compile Include="Base\BacnetDateRange.cs" />
     <Compile Include="Base\BacnetDeviceObjectPropertyReference.cs" />
     <Compile Include="Base\BacnetError.cs" />
-    <Compile Include="Base\BacnetEventNotificationData.cs" />
+    <Compile Include="Base\EventNotification\EventValues\EventValuesBase.cs" />
+    <Compile Include="Base\EventNotification\EventValues\ChangeOfLifeSafety.cs" />
+    <Compile Include="Base\EventNotification\EventValues\UnsignedRange.cs" />
+    <Compile Include="Base\EventNotification\EventValues\ChangeOfState.cs" />
+    <Compile Include="Base\EventNotification\EventValues\OutOfRange.cs" />
+    <Compile Include="Base\EventNotification\EventValues\FloatingLimit.cs" />
+    <Compile Include="Base\EventNotification\EventValues\ChangeOfValue.cs" />
+    <Compile Include="Base\EventNotification\EventValues\BufferReady.cs" />
+    <Compile Include="Base\EventNotification\EventValues\ChangeOfBitString.cs" />
+    <Compile Include="Base\EventNotification\StateTransition.cs" />
+    <Compile Include="Base\EventNotification\EventNotificationData.cs" />

```

Comment:
```EventNotification``` - > ```Notifications``` to make it shorter? and maybe not introduce sub directory/namespace ```EventValues``` since there will be no more classes and we don't need to sparate them?

Reply:
Well, there are ChangeOfValueNotifications too which are something completely different... that's why I went with a more specific name.

As to EventValues... if we don't have the namespace, wouldn't we "have to" rename the types? EventNotification.OutOfRange conveys the wrong purpose, it should then be EventNotification.OutOfRangeEventValues - which makes things worse, IMHO.

Reply:
I'm only thinking out loud, making sure we don't clutter the code with unnecessary or unnecessarily long namespaces. The same for class names, if the namespace already says EventNotification, maybe we could skip 'event' part in class names. But we could also introduce changes later when we feel like this is bothering us.